src/client/lazy-app/Compress/index.tsx

```diff
@@ -257,12 +262,16 @@ function processorStateEquivalent(a: ProcessorState, b: ProcessorState) {
   return true;
 }
 
+const loadingIndicator = '⏳ ';
+
 const originalDocumentTitle = document.title;
 
-function updateDocumentTitle(filename: string = ''): void {
-  document.title = filename
-    ? `${filename} - ${originalDocumentTitle}`
-    : originalDocumentTitle;
+function updateDocumentTitle(loadingFileInfo: LoadingFileInfo): void {
+  const { loading, filename } = loadingFileInfo;
+  document.title =
+    filename && loading
+      ? `${filename} - ${originalDocumentTitle}`
+      : originalDocumentTitle;
```

Comment:
Since this function is formatting the document title, it should take care of the loading indicator. It shouldn't be part of the filename.

```suggestion
  let title = '';
  if (loading) title += loadingIndicator;
  if (filename) title += filename + ' - ';
  title += originalDocumentTitle;
  document.title = title;
```